os/drivers/audio/alc5658.c

```diff
@@ -597,6 +597,21 @@ static int alc5658_configure(FAR struct audio_lowerhalf_s *dev, FAR const struct
 				ret = -EDOM;
 		}
 		break;
+
+		case AUDIO_FU_MUTE: {
+			/* Mute or unmute:  true(1) or false(0) */
+
+			bool mute = caps->ac_controls.hw[0];
+			auddbg("    mute: 0x%x\n", mute);
+
+			if (false == mute || true == mute) {
+					priv->mute = mute;
+					alc5658_setvolume(priv);
+				}
+			else
```

Comment:
kindly use the flower brackets for empty else case as well. < coding guidelines >

Reply:
Additionally, 
1. line 610, need to remove one tab
2. else should be at next of closing bracket
3. As Pradeep said, even though else has a single statemen, you recommand using brackets.